### PR TITLE
[#39] Fix Flake8 in 01_board_and_wall.py

### LINKED ISSUE #39: Run flake8 on the project
after installing flake8 it reveals a lot of linting issues in the code.
Since the PR's will take a lot of lines from a lot of files just open the issues regarding this.
When flake8 passes this issue will be closed

steps to reproduce:
- install dev dependencies with `pipenv install --dev`
- run `flake8 .` at top level

### PR DESCRIPTION
**Pull request by f-osorio**
Fixes #39

Flake8 Update for examples/01_board_and_wall.py

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: examples/01_board_and_wall.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 
-import examples_includes
```
Comment (arnauddupuis) [inline examples/01_board_and_wall.py]: Why did you remove that line?

Comment (kaozdl) [reply examples/01_board_and_wall.py]: if it's not used i think there is no need.
flake8 will trhow an errror for unused imports

Comment (arnauddupuis) [reply examples/01_board_and_wall.py]: Actually it is only 2 lines to include ../ into the path so when you run the examples from the example directory without installing the library first, it still work.
Maybe it's the wrong way of doing it.
